src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/preprocessing.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_plugin/preprocessing.hpp"
+
+#ifdef ENABLE_GAPI_PREPROCESSING
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+const std::vector<ov::element::Type> inputPrecisions = {ov::element::u16, ov::element::f32};
+
+const std::vector<ov::AnyMap> configs = {{}};
+
+INSTANTIATE_TEST_SUITE_P(BehaviourPreprocessingTestsViaSetTensor,
```

Comment:
I suppose `BehaviourPreprocessingTestsViaSetTensor` are obsolete tests which should be removed with G-API preprocessing.
In OV 2.0 we cannot apply preprocessing together with SetTensor API, because now it's a part of the model

Reply:
@ilya-lavrenov so you mean we can not upgrade this test & will remove it in 24.0, right?

Reply:
yes